log/handler.go

```diff
@@ -277,13 +277,13 @@ func builtinReplace(_ []string, attr slog.Attr) slog.Attr {
 		if v == nil {
 			attr.Value = slog.AnyValue("<nil>")
 		} else {
-			attr.Value = slog.AnyValue(formatLogfmtBigInt(v))
+			attr.Value = slog.AnyValue(v.String())
```

Comment:
It seems to me that for some reason, the case of `*big.Int` is not needed? Not quite sure why. Perhaps `Stringer` ? Try dropping this clause, it still works.

Reply:
We explicitly return `slog.StringValue` now.